build.gradle

```diff
@@ -46,36 +46,111 @@ configurations {
                 url 'https://foursquare.jfrog.io/foursquare/libs-release/'
             }
         }
+        
+        exclude group: 'com.android.support', module: 'animated-vector-drawable'
+        exclude group: 'com.android.support', module: 'appcompat-v7'
+        exclude group: 'com.android.support', module: 'asynclayoutinflater'
+        exclude group: 'com.android.support', module: 'collections'
+        exclude group: 'com.android.support', module: 'coordinatorlayout'
+        exclude group: 'com.android.support', module: 'cursoradapter'
+        exclude group: 'com.android.support', module: 'customview'
+        exclude group: 'com.android.support', module: 'documentfile'
+        exclude group: 'com.android.support', module: 'drawerlayout'
+        exclude group: 'com.android.support', module: 'interpolator'
+        exclude group: 'com.android.support', module: 'loader'
+        exclude group: 'com.android.support', module: 'localbroadcastmanager'
+        exclude group: 'com.android.support', module: 'print'
+        exclude group: 'com.android.support', module: 'recyclerview-v7'
+        exclude group: 'com.android.support', module: 'slidingpanelayout'
+        exclude group: 'com.android.support', module: 'support-annotations'
+        exclude group: 'com.android.support', module: 'support-compat'
+        exclude group: 'com.android.support', module: 'support-core-ui'
+        exclude group: 'com.android.support', module: 'support-core-utils'
+        exclude group: 'com.android.support', module: 'support-fragment'
+        exclude group: 'com.android.support', module: 'support-vector-drawable'
+        exclude group: 'com.android.support', module: 'swiperefreshlayout'
+        exclude group: 'com.android.support', module: 'transition'
+        exclude group: 'com.android.support', module: 'versionedparcelable'
+        exclude group: 'com.android.support', module: 'viewpager'
+        
         resolutionStrategy.eachDependency { details ->
-            if (details.requested.name == 'runtime') {
-                details.useTarget group: details.requested.group, name: details.requested.name, version: details.requested.version
-                
+            if (details.requested.group == 'androidx.activity') {
```

Comment:
im not super familiar with unity and dont remember when we set this up, but it feels wrong to have to do all of this.

Reply:
for some reason it downloads multiple versions of some deps, maybe trying that jettifier will fix this too